tests/test_execution_compute_plan.py

```diff
@@ -0,0 +1,257 @@
+import substra
+
+import pytest
+
+import substratest as sbt
+
+
+@pytest.mark.skip('may raise MVCC errors')
+def test_compute_plan(factory, session_1, session_2):
```

Comment:
that's the pre-existing test, moved from `test_execution.py`

Reply:
👍